project_manager/project_manager/views.py

```diff
@@ -1,6 +1,6 @@
 from django.contrib.auth import authenticate, login
-from django.shortcuts import render, render_to_response, RequestContext, HttpResponseRedirect
-
+from django.shortcuts import render, render_to_response, HttpResponseRedirect
+from django.template import RequestContext
```

Comment:
Did it work in Django 1.11 after this change?

Reply:
yes,it will work.